### PR TITLE
feat: History & UserProfile

### PR DESCRIPTION
**Pull request by himadri2110**
- Add video to History, delete the video from History, clear history
- User Profile page

### CODE REVIEW

File: src/pages/UserProfile/UserProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  return (
+    <section className="main-section">
+      <Sidebar />
+
+      <div className="component-container profile-container">
+        <div className="profile-details">
+          <img
+            src={`https://avatars.dicebear.com/api/initials/${user.firstName}%20${user.lastName}.svg?b=%238553fa&r=50&size=80&fontSize=40`}
+            alt={fullName}
```
Comment (BelideNikhil) [inline src/pages/UserProfile/UserProfile.jsx]: you can extract this into utils so that the code becomes clean!

**Review by BelideNikhil — APPROVED** [review]
The UI works as it should,
The code is well structured and readable
GTG!

File: src/pages/UserProfile/UserProfile.jsx (hunk 1 above)
Comment (himadri2110) [reply src/pages/UserProfile/UserProfile.jsx]: Okay, I'll do that. Thanks.

**Review by Shrinkhlarajpoot — APPROVED** [review]
Looks good..
GTG.

**Pull request merged**